image/elasticsearch.yml

```diff
@@ -0,0 +1,16 @@
+cluster.name: "docker-cluster"
+network.host: 0.0.0.0
+
+# minimum_master_nodes need to be explicitly set when bound on a public IP
+# set to 1 to allow single node clusters
+# Details: https://github.com/elastic/elasticsearch/pull/17288
+discovery.zen.minimum_master_nodes: 1
```

Comment:
ありがとうございます 🙏 

設定値で質問ですが、`indices.fielddata.cache.size ` を設定した場合、`indices.breaker.fielddata.limit（default=60%）` より下げたほうが良さそうです

https://www.elastic.co/guide/en/elasticsearch/guide/current/_limiting_memory_usage.html#fielddata-size
> If the circuit-breaker limit is lower than the cache size, no data will ever be evicted. In order for it to work properly, the circuit breaker limit must be higher than the cache size.

なので、ブログ記事にある通り75%ではなく、40%くらいですかね？

類似例：http://evertrue.github.io/blog/2014/11/16/3-performance-tuning-tips-for-elasticsearch/

> For us, since we specify indices.fielddata.cache.size, we also need to be aware of indices.breaker.fielddata.limit. It defaults to 60% which means our cluster will start throwing exceptions even though there is technically plenty of memory on the box to serve the request. Remember, we have our field data size set to 75%, well above the default circuit breaker limit.